### ISSUE TITLE
Vue composition api

### ISSUE THREAD

**Issue by lukasbrzobohaty** [iFaxity/vite-plugin-istanbul#178]
Hi,

I have a some issue with instrument code by istanbul in project with vue and composition api. 
When I build project then istanbul create three coverage functions with the same path in dist folder. 

First function is corect, it is for code in `<script setup>`. Second function has empty coverage and it is for `<template>`. The last one is for `<style>`.

Probably browser take only the last function with empty coverage. Then coverage don't work for these components.

First function:
![Snímek obrazovky pořízený 2024-02-13 11-33-47](https://github.com/iFaxity/vite-plugin-istanbul/assets/83454835/f7a6eb02-c79d-4d15-9fe0-df960d7400f2)

Second function:
![Snímek obrazovky pořízený 2024-02-13 11-34-43](https://github.com/iFaxity/vite-plugin-istanbul/assets/83454835/dd0b3873-243d-4d35-8a57-297ee0993856)

Third function:
![Snímek obrazovky pořízený 2024-02-13 11-35-17](https://github.com/iFaxity/vite-plugin-istanbul/assets/83454835/30252ef0-590d-4760-ba7b-b4f6db577073)

Browser coverage:
![Snímek obrazovky pořízený 2024-02-13 11-35-42](https://github.com/iFaxity/vite-plugin-istanbul/assets/83454835/efad7af0-a0e4-4486-81e2-03b04f529af8)

### Reproduction

https://github.com/lukasbrzobohaty/vite-istanbul

### Steps to reproduce

1. `npm install`
2. `npm run build`
3. `npm run start`
4. visit http://localhost:8080, show `window.__coverage__` in console and look to `.../src/components/HelloWorld.vue
`

### System Info

```shell
System:
    OS: Linux 6.5 Linux Mint 21 (Vanessa)
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 16.26 GB / 31.15 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 18.8.0 - ~/.nvm/versions/node/v18.8.0/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 8.18.0 - ~/.nvm/versions/node/v18.8.0/bin/npm
  Browsers:
    Chrome: 121.0.6167.160
  npmPackages:
    @vitejs/plugin-vue: ^5.0.3 => 5.0.4 
    vite: ^5.1.0 => 5.1.1 
    vite-plugin-istanbul: ^5.0.0 => 5.0.0
```

### Used Package Manager

npm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/iFaxity/vite-plugin-istanbul/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't [already an issue](https://github.com/iFaxity/vite-plugin-istanbul/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a vite-plugin-istanbul issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to [vuejs/core](https://github.com/vuejs/core) instead.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by Nol-go** [comment]
duplicate #96 (I think)

**Comment by iFaxity** [comment]
Hi @lukasbrzobohaty,

could you please try the new rc version released with #180 if this fixes your issue?

**Comment by iFaxity** [comment]
Closing due to inactivity.
